src/components/list-lemplate/table-template.tsx

```diff
@@ -66,21 +90,8 @@ const TableTemplate = <T extends { id: Key }>({
             };
         });
 
-        if (isAdmin) {
-            columns.push({
-                title: "",
-                dataIndex: "",
-                key: "x",
-                width: "5%",
-                render: (record: { id: string }) => (
-                    <ActionsMenu
-                        pageName={pageName}
-                        onDelete={menuProps.onDelete}
-                        onUpdate={menuProps.onUpdate}
-                        record={record}
-                    />
-                ),
-            });
+        if (isAdmin || pageName === t("cvs.cv") || pageName === t("users.user")) {
```

Comment:
"Projects" page could use something like this too ("project" (details) option probably got grayed out in the example because details page doesn't really exist there)
![image](https://github.com/savelievamareena/cv-final/assets/99599598/f4dc3143-b811-4930-aafe-a45f6101dee0)

Reply:
add link to project page